target_chains/ton/contracts/wrappers/PythTest.ts

```diff
@@ -99,8 +104,8 @@ export class PythTest implements Contract {
     });
   }
 
-  async getPriceUnsafe(provider: ContractProvider, priceFeedId: HexString) {
-    const result = await provider.get("test_price_unsafe", [
+  async getGetPriceUnsafe(provider: ContractProvider, priceFeedId: HexString) {
```

Comment:
the function name here might sound weird but this is technically correct, it follows a convention of `{get/send}{test_name)`

Reply:
This doesn't make much sense to me. isn't it up to us to define function names here? If the convention is {get/send}{test_name} i assume the convention on the actual method names is to be without get/send prefix so maybe remove it from there?